src/split-panel/implementation.tsx

```diff
@@ -185,77 +185,71 @@ export const SplitPanelImplementation = React.forwardRef<HTMLElement, SplitPanel
      * is still needed for the early return to prevent execution
      * of the following code.
      */
-    if (isRefresh && !isOpen && position === 'side') {
+    if (isRefresh && !isAppLayoutToolbarEnabled() && !isOpen && position === 'side') {
       return <></>;
     }
 
     return (
-      <Transition in={isOpen ?? false}>
-        {(state, transitioningElementRef) => (
```

Comment:
This change impacts even the classic design? is it expected too?

Reply:
Classic isn't expected to have any animations. The [current classes](https://github.com/cloudscape-design/components/blob/main/src/split-panel/motion.scss#L9-L21) only apply to refresh and I also checked manually that this was the case. It shouldn't have any impact.